test/template/template_test.go

```diff
@@ -40,9 +40,8 @@ func TestChartTemplateNoLogCollection(t *testing.T) {
 	// render the tempate
 	output := helm.RenderTemplate(t, options, helmChartPath, releaseName, []string{"templates/statefulset.yaml"})
 
```

Comment:
This template test should not be included. It will mess up the commit history. You should probably rebase first, and make sure the PR contains change only relate with the commit message, which is CLD-700.

Reply:
Fixed.